src/main/java/com/uber/cadence/serviceclient/WorkflowServiceTChannel.java

```diff
@@ -355,7 +356,14 @@ public void close() {
   }
 
   private <T> T measureRemoteCall(String scopeName, RemoteCall<T> call) throws TException {
+    return measureRemoteCallWithTags(scopeName, call, null);
+  }
+
+  private <T> T measureRemoteCallWithTags(String scopeName, RemoteCall<T> call, Map<String, String> tags) throws TException {
     Scope scope = options.getMetricsScope().subScope(scopeName);
+    if (tags != null){
```

Comment:
nit: missing space before `{`

Reply:
Super good catch! Lol looks like I forgot to ocnfigure my autosaving plugin